src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -316,10 +263,12 @@ export class HuiCardPicker extends LitElement {
   }
 
   private _cardPicked(ev: Event): void {
-    const config: LovelaceCardConfig = (ev.currentTarget! as CardPickTarget)
-      .config;
+    const target = ev.currentTarget! as CardPickTarget;
 
-    fireEvent(this, "config-changed", { config });
+    fireEvent(this, "config-changed", {
+      config: target.config,
+      documentationURL: target.documentationURL,
```

Comment:
Let's not pass this but always use the same function to get the documentation url